src/main/java/nl/tudelft/scrumbledore/userinterface/GameDisplay.java

```diff
@@ -355,10 +346,8 @@ private static void renderStatic() {
   private static void renderDynamic() {
     dynamicContext.clearRect(0, 0, Constants.GUIX, Constants.GUIY);
 
-    renderPlayer();
-    renderBubbles();
-    renderNPC();
-    renderFruit();
+    ArrayList<LevelElement> dynamicElements = currentGame.getCurrentLevel().getDynamicElements();
+    renderLevelElements(dynamicElements, dynamicContext);
```

Comment:
Could also be constructed as:
renderLevelElements(currentGame.getCurrentLevel().getDynamicElements(), dynamicContext);